### PR TITLE
add zumper-angular-payments@1.0.7 w/ git auto-update

### LINKED ISSUE #9811: [Request] Add Angular Payments
**Library name: Angular Payments** 
**Git repository url: https://github.com/zumper/angular-payments**
**npm package url(optional): https://www.npmjs.com/package/angular-payments** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
Pull request for issue: #9811 
Related issue(s): # #
@pvnr0082t please help me review this PR, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/zumper/angular-payments
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/angular-payments
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 4
   - Count of watchers: 29
   - Count of forks: 254
 * NPM download stats (optional):
   - Downloads in the last day: 106
   - Downloads in the last week: 520
   - Downloads in the last month: 2499

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9811, cc @jribble, @laurihy, @atav32

### CODE REVIEW

File: ajax/libs/angular-payments/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "author": {
+    "name": "Lauri Hynynen",
+    "url": "https://github.com/laurihy"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/zumper/angular-payments.git"
+  },
+  "license": "MIT",
+  "npmName": "angular-payments",
```
Comment (pvnr0082t) [inline ajax/libs/angular-payments/package.json]: There is only one version on NPM but 8 on GitHub. It's better to use git auto-update config.

Comment (PeterDaveHello) [inline ajax/libs/angular-payments/package.json]: Nice catch!

**Comment by kennynaoh** [comment]
@pvnr0082t I have changed to the git auto-update. Please help me review it again. Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
The name should be `zumper-angular-payments`, it's not an origin repo.

**Comment by kennynaoh** [comment]
@pvnr0082t I have changed the name. Please help me review it again. Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**